std_msgs/String[] goals
---
std_msgs/Bool response